src/test/integration/IntegrationBase.t.sol

```diff
@@ -104,6 +106,17 @@ abstract contract IntegrationBase is IntegrationDeployer {
         numOperators++;
         return (operator, strategies, tokenBalances);
     }
+    
+    function _newRandomAVS(IStrategy[] memory strategies) internal returns (AVS avs, uint32 operatorSetId) {
+        string memory avsName = string.concat("avs", numAVSs.toString());
```

Comment:
From my perspective this isn't a "random" AVS since we're passing in the strategies. What makes a random AVS is creating a random number of operatorSets with a random set of strategies in each operatorSet

Reply:
fixed in [d5c877e](https://github.com/Layr-Labs/eigenlayer-contracts/pull/894/commits/d5c877ed185e4782f6a7c3c5ffb55dc0a5a45031)